Helper/Config.php

```diff
@@ -330,6 +335,19 @@ public function getSuccessPageRedirect()
         );
     }
 
+     /**
+     * Get Custom javascript function call on success
+     *
+     * @return  string
+     */
+    public function getjavascriptSuccess()
+    {
+        return $this->scopeConfig->getValue(
```

Comment:
not sure how these config is stored; do we need to escape/unescape string? I remember seeing other plugin cause issue if you store string like "{", "(" as is.

Reply:
I already test some analytics and JavaScript functions.
there is no issue.